ls1intum/Artemis
# `Development`: Document MySQL to PostgreSQL database conversion

### Checklist
#### General



### Motivation and Context
The previous conversion steps resulted in a broken PostgreSQL database and had to be removed (#6251). This PR adds a working database migration guide.

### Description
- Mention PostgreSQL in the regular setup documentation.
- Add steps and scripts for conversion from MySQL to PostgreSQL databases to the admin documentation.

### Steps for Testing
- Check that conversion steps are clear, contain no typos, and are [rendered correctly](https://add-readthedocs-link.todo).
- Optional manual tests:
  - Apply the conversion to your local MySQL database.

### Review Progress






#### Code Review
#### Manual Tests
